helm/nessie/values.yaml

```diff
@@ -171,79 +171,88 @@ catalog:
+        # https://docs.aws.amazon.com/AmazonS3/latest/userguide/using-access-points.html.
+        accessPoint: ~
+        # -- Authorize cross-region calls when contacting an access point. The default is false.
+        allowCrossRegionAccessPoint: false
+        # -- Controls the authentication mode for Catalog clients accessing this bucket. Valid values
+        # are:
+        # - REQUEST_SIGNING: Each client I/O request is individually authorized (signed) by the
+        # Catalog server. This is the default.
+        # - ASSUME_ROLE: Clients receive session credentials (according to the role and IAM policy
+        # from bucket configuration) for the whole duration of client sessions.
+        clientAuthenticationMode: ~  # REQUEST_SIGNING
+
+        # -- Settings only relevant when clientAuthenticationMode is ASSUME_ROLE.
+        assumeRole:
+          # -- The STS endpoint. Optional; if not provided, the default is used. This parameter must
+          # be set if the cloud provider is not AMAZON and the catalog is configured to use S3
+          # sessions (e.g. to use the "assume role" functionality).
+          stsEndpoint: ~  # "https://sts.amazonaws.com"
+          # -- The ARN of the role to assume for accessing S3 data. This parameter is required for
+          # Amazon S3, but may not be required for other storage providers (e.g. Minio does not use it
+          # at all).
+          roleArn: ~  # "arn:aws:iam::123456789012:role/role-name"
+          # -- The IAM policy in JSON format to be used as an inline session policy. Optional.
+          sessionIamPolicy: ~  # "{ \"Version\": \"2012-10-17\", \"Statement\": [ { \"Effect\": \"Allow\", \"Action\": \"s3:*\", \"Resource\": \"*\" } ] }"
+          # -- An identifier for the assumed role session. This parameter is most important in cases
+          # when the same role is assumed by different principals in different use cases.
+          roleSessionName: ~  # nessie
+          # -- An identifier for the party assuming the role. This parameter must match the external
+          # ID configured in IAM rules that govern the assume role process for the specified roleArn.
+          externalId: ~
+          # -- A higher bound estimate of the expected duration of client "sessions" working with data in
+          # this bucket. A session, for example, is the lifetime of an Iceberg REST catalog object on
+          # the client side. This value is used for validating expiration times of credentials
+          # associated with the warehouse. If unset, a default of one hour is assumed.
+          clientSessionDuration: ~
+          # -- The time period to subtract from the S3 session credentials (assumed role credentials)
+          # expiry time to define the time when those credentials become eligible for refreshing.
+          # Not overridable on a per-bucket basis. The default is PT5M (5 minutes).
+          sessionCredentialRefreshGracePeriod: ~  # PT5M
```

Comment:
These last 3 options are not overridable per bucket. Maybe move them to `transport` or some other section outside `defaultOptions`?

Reply:
Moved to a new section